packages/sirius-web/backend/sirius-web-infrastructure/src/main/resources/db/changelog/2026.1/01-drop-fk-project-id-in-project-semantic-data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2025 Obeo.
+  ~ This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v2.0
+  ~ which accompanies this distribution, and is available at
+  ~ https://www.eclipse.org/legal/epl-2.0/
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+  ~
+  ~ Contributors:
+  ~     Obeo - initial API and implementation
+  -->
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="01-drop-fk-project-id-in-project-semantic-data-table" author="ntinsalhi">
+        <dropForeignKeyConstraint
+                baseTableName="project_semantic_data"
+                constraintName="fk_project_semantic_data_project_id"/>
+    </changeSet>
```

Comment:
I will also need to see the foreign key between project_semantic_data and semantic_data being deleted in this first contribution. You will have to update the name of this changeset to be a bit more generic then.